wiki/release/release-process.md

```diff
@@ -333,7 +341,7 @@ select the staging repository associated with the RC candidate that was approved
 will be like `orgapachepulsar-XYZ`. Select the repository and click on "Release". Artifacts
 will now be made available on Maven central.
 
-## 10. Publish Docker Images
+## 9. Publish Docker Images
```

Comment:
Maybe we should add the list of committer with access to apachepulsar org?

Reply:
I don't know where can I found these committers. Could you provide any help here? @codelipenghui